### DISCUSSION TITLE
How to make videojs wait to load media until load() function is called?

### DISCUSSION THREAD

**Discussion by raisonjoseph** [videojs/video.js#7435] [Q&A]
I want to make player to wait fetching the media file until an event or triggered manually.

**Comment by gkatsev** [comment]
You probably want to set `preload` to `none`. Unfortunately, if you're playing back HLS or DASH via [videojs http-streaming](https://github.com/videojs/http-streaming) (VHS), `player.load()` won't work right now. However, with `preload=none` we'll wait until playback is requested before starting to load things.
